eclcli/sss/v1/user.py

```diff
@@ -78,13 +78,15 @@ def get_parser(self, prog_name):
             metavar='<boolean>',
             type=utils.parse_bool,
             choices=[True, False],
-            help='If this flag is set true, notification eamil will be sent to new users email.'
+            help='If this flag is set true, notification email will be sent to new users email.'
         )
         parser.add_argument(
             '--password',
             metavar='<password>',
             help='Initial password of new user. If this parameter is not designated, '
-                 'random initial password is generated and applied to new user.'
+                 'random initial password is generated and applied to new user. '
+                 'Specify a minimum of 8 characters and include at least one each of ' 
+                 'uppercase and lowercase letters and numbers.'
```

Comment:
```suggestion
                 'uppercase, lowercase letters, and numbers.'
```
こちらの方が自然な英文になると感じました。